src/sentry/apidocs/parameters.py

```diff
@@ -237,6 +237,7 @@ class ProjectParams:
 - `legacy-browser`: Filter out known errors from legacy browsers. Older browsers often give less
 accurate information, and while they may report valid issues, the context to understand them is
 incorrect or missing.
+- `filtered-transaction`: Filter out transactions for healthcheck and ping endpoints.
```

Comment:
How is filtered-transactions toggled? If it's done using the `active` query param, you must add it to the [active strin explanation](https://github.com/getsentry/sentry/blob/master/src/sentry/apidocs/parameters.py#L248). Also this filter appears to be set behind a feature flag so is it available to all users by default?

<img width="551" alt="Screenshot 2023-06-30 at 12 25 06 PM" src="https://github.com/getsentry/sentry/assets/67301797/7abf8a69-037d-497a-9cfe-42d090925c68">